_posts/2017-04-26-Relay-Mutations.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: post_longform
+title: Relay Mutations
+date: 2017-04-26
+categories: [relay, graphql, JavaScript, guest]
+author: nik
+---
+
+> Hey there everyone, it took us two years to [make our](https://github.com/artsy/metaphysics/pull/583) GraphQL implementation support any mutations and we still don't have any Relay mutations yet.
+>
+> Last month, I talked with the team at [Graph.cool](https://www.graph.cool/) about having them talk through Relay mutations comprehensively as a guest post on the Artsy Engineering blog. So, I'm really excited to introduce this great post on the topic by [Nikolas Burk](http://twitter.com/nikolasburk).
+>
+> -- Orta
+
+#  The Magic behind Relay Mutations
+
+[Relay](https://facebook.github.io/relay/) is a powerful GraphQL client for React and React Native applications. It was open sourced by Facebook alongside GraphQL in 2015 and is a great tool for supporting you with managing your app's data layer.
+
+In this post, we are going to explore how Relay mutations work by the example of a React Native app, the code can be found on [GitHub](https://github.com/graphcool-examples/react-native-relay-pokedex-example). Our sample application is a simple _Pokedex_, where users can manage their Pokemons.
```

Comment:
The word 'by' feels a bit awkward, maybe "..how Relay mutations work by looking at a React Native app. The code can be found on ..."